runtime/compiler/runtime/RuntimeAssumptions.hpp

```diff
@@ -229,6 +229,10 @@ class TR_AddressSet
       _maxAddressRanges(maxAddressRanges)
       {}
 
+   void destroy();
+   void getRanges(std::vector<TR_AddressRange> &ranges);
+   void setRanges(std::vector<TR_AddressRange> &ranges);
```

Comment:
As mentioned elsewhere the `ranges` vector could be marked `const` for `setRanges`